rust/opendp-ffi/src/util.rs

```diff
@@ -37,16 +39,16 @@ impl Type {
         let id = TypeId::of::<T>();
         // First try to find a registered type (which will have the nice descriptor). In lieu of that, create one on the fly.
         let type_ = TYPE_ID_TO_TYPE.get(&id);
-        type_.map_or_else(|| Self::of_unregistered::<T>(), Clone::clone)
+        type_.map_or_else(Self::of_unregistered::<T>, Clone::clone)
```

Comment:
Interesting, didn't know you could get a function arg from a generic this way, thanks.